### ISSUE TITLE
Short-term archive is broken on maint-2.0

### ISSUE THREAD

**Issue by tangq** [E3SM-Project/E3SM#5034]
Short-term archive seems broken on branch maint-2.0.

Running `./case.st_archive` (hash: `6275091262780c7a4b54ba56dcfa013ab1efa149`) gets errors:
```
Traceback (most recent call last):
  File "./case.st_archive", line 22, in <module>
    from standard_script_setup          import *
ModuleNotFoundError: No module named 'standard_script_setup'
```

**Comment by rljacob** [comment]
@tangq is it broken on master?

**Comment by tangq** [comment]
I only tried maint-2.0, so am not sure about master.

**Comment by jasonb5** [comment]
@jgfouca @tangq Looks like https://github.com/E3SM-Project/E3SM/pull/4940 didn't get merged into maint-2.0.

**Comment by jgfouca** [comment]
It looks like this has been taken care of.
